packages/db/index.js

```diff
@@ -180,7 +180,7 @@ export class DBClient {
       throw new DBError(error)
     }
 
-    return count > 0
+    return count === 1
```

Comment:
Would it be better to throw an error if there's more than one row returned? Otherwise, currently, if there's more than one row set to `true`, it will return `false`.

I'd actually be tempted to make a generic `getUserTagValue` function which you pass `'HasAccountRestriction'` or `'HasPsaAccess'` (or whatever) into, and it does the query and the count check, and then returns the value.

Reply:
Added the generic function and left the two existing ones as wrappers to check the type. Thanks!